rtCamp/web-stories-wp
# Removes Standalone blocks in favour of one master block.

## Summary

This PR merges Latest Stories Block and Selected Web Stories block into one `Web Stories List` block.

## Relevant Technical Choices



## To-do

- Work on minor details and performance improvisation.

## User-facing changes

- 'Latest Web Stories' and 'Selected Web Stories' have been removed from the insert menu.
- 'Web Stories List' replaces the two blocks.
- User will be presented with the configuration screens to select the type of block and view for the block upon insertion.

## Testing Instructions

- Try to insert the Web Stories block, observe that 'Latest Web Stories' and 'Selected Web Stories' blocks are not listed anymore.
- Find the 'Web Stories List' block in the insertion menu.
- Insert the 'Web Stories List' block and select appropriate options for the block type and view type.

---


Also following PRs are part of this PR #127 , #128 

Fixes #93
